extensions/amp-list/0.1/amp-list.js

```diff
@@ -298,12 +307,13 @@ export class AmpList extends AMP.BaseElement {
 
   /**
    * @private
+   * @return {!Promise}
    */
   maybeResizeListToFitItems_() {
     if (this.loadMoreEnabled_) {
-      this.attemptToFitLoadMore_(dev().assertElement(this.container_));
+      return this.attemptToFitLoadMore_(dev().assertElement(this.container_));
```

Comment:
Optional: Do we ever use this return value? If not, it might be simpler (if less consistent) to leave `attemptToFitLoadMore_` and its siblings as void return. E.g. readers won't need to trace the return value when reading `attemptToFitLoadMore_` and siblings.